Simple Inventory Management System/Inventory.cs

```diff
@@ -28,5 +28,40 @@ public void DisplayProducts()
             }
         }
 
+        public void EditProduct(string productName)
+        {
+            Product product = products.Find(p => p.Name == productName);
```

Comment:
I have notices that you are not using `var`, it is a better practice to use `var` instead of class name